### PR TITLE
Wired up dashboards to databases

### LINKED ISSUE #36: Add getting buildings from database for ManagerDashboard, OwnerDashboard, WorkerDashboard view.

### PR DESCRIPTION
**Pull request by kwiniarski97**
Closes #36

### CODE REVIEW

File: projekt/Budowa/src/main/java/org/budowa/flow/worker/WorkerDashboardScene.java [hunk 1]
```diff
@@ -38,8 +39,8 @@ public void handleRefresh(ActionEvent actionEvent) {
     /* Methods [protected] */
 
     @Override
-    protected Collection<Building> loadBuildings() {
-        // todo: get user id from session manager
-        return this.buildingsService.getWorkerBuildings(0);
+    protected Building[] loadBuildings() {
+        var userId = this.sessionManager.getUser().getId();
```
Comment (FilipAlbrycht) [inline projekt/Budowa/src/main/java/org/budowa/flow/worker/WorkerDashboardScene.java:43]: typing?

File: projekt/Budowa/src/main/java/org/budowa/repositories/BuildingsRepository.java [hunk 2]
```diff
@@ -70,23 +74,30 @@ public int insert(Building building){
...
+    public void delete(Building building) {
         em.getTransaction().begin();
         em.remove(building);
         em.getTransaction().commit();
     }
 
 
+    public Building[] getWorkerBuildings(int userId) {
+        Query query = em.createNativeQuery("select * from buildings right join workers_buildings wb on buildings.id = wb.building_id where wb.user_id =" + userId, Building.class);
+        var list = query.getResultList();
```
Comment (FilipAlbrycht) [inline projekt/Budowa/src/main/java/org/budowa/repositories/BuildingsRepository.java:100]: as far as I know, there is possibility to use generic type over here

File: projekt/Budowa/src/main/java/org/budowa/flow/worker/WorkerDashboardScene.java (hunk 1 above)
Comment (kwiniarski97) [reply projekt/Budowa/src/main/java/org/budowa/flow/worker/WorkerDashboardScene.java:43]: could you elaborate?

File: projekt/Budowa/src/main/java/org/budowa/repositories/BuildingsRepository.java (hunk 2 above)
Comment (kwiniarski97) [reply projekt/Budowa/src/main/java/org/budowa/repositories/BuildingsRepository.java]: done

File: projekt/Budowa/src/main/java/org/budowa/flow/worker/WorkerDashboardScene.java (hunk 1 above)
Comment (FilipAlbrycht) [reply projekt/Budowa/src/main/java/org/budowa/flow/worker/WorkerDashboardScene.java:43]: user's id should be number so type it as int

Comment (kwiniarski97) [reply projekt/Budowa/src/main/java/org/budowa/flow/worker/WorkerDashboardScene.java:43]: it is a number. var is an anonymous type. the compiler is able to intercept a type of the variable to it is redundant to type a class name or a primitive type. https://javastart.pl/baza-wiedzy/slownik/slowo-kluczowe-var

**Pull request merged**